docs/reference/inference/put-inference.asciidoc

```diff
@@ -210,11 +220,37 @@ allocations. Must be a power of 2. Max allowed value is 32.
 Settings to configure the {infer} task. These settings are specific to the
 `<task_type>` you specified.
 +
+.`task_settings` for the `completion` task type
+[%collapsible%closed]
+=====
+`user`:::
+(Optional, string)
+For `openai` service only. Specifies the user issuing the request, which can be
+used for abuse detection.
+=====
++
+.`task_settings` for the `rerank` task type
+[%collapsible%closed]
+=====
+`return_documents`::
+(Optional, boolean)
+For `cohere` service only. Specifies whether to return results with the doc
+text. If `false`, the API returns a list of `index` and `relevance score` value
+where `index`` is inferred from the doc list passed into the request. If `ture`, 
+the API returns an ordered list of `index`, `text` and `relevance score` values
+where `index` and `text` refers to the doc list passed into the request.
```

Comment:
I'm not sure that we need the sentences "If `false` ...", "If `true`...". I think that just the previous sentence could be clear enough. We could maybe update that sentence to say "Specify whether to return doc text within the results.".